src/screens/History/History.jsx

```diff
@@ -9,16 +9,16 @@ const History = () => {
     return (
         <div className='container history-elem-container'>
             <div className='row'>
-                <Link to='/' className='ml-5 mt-4 rounded-circle arrow-back-elem'><FontAwesomeIcon icon={faChevronLeft} size="lg" /></Link>
+                <Link to='/' className='ml-4 mt-4 rounded-circle arrow-back-elem'><FontAwesomeIcon icon={faChevronLeft} size="lg" /></Link>
             </div>
             <div className='row'>
                 <h1 className='text-center history-text-elem'>Histórico</h1>
             </div>
-            <div className='row'>
-                <div className="col">
-                  <DateRangePickers text='Desde' />
+            <div className='row date-range-pickers-container'>
+                <div className="col p-0">
+                  <DateRangePickers className='pt-3' text='Desde' />
```

Comment:
You're giving unnecessary context to the component you've built. You have created a "date picker component". Here you're using it as a range picker (range picker is the combination of the two date pickers). But it could also be used as a single DatePicker. In that last case, the naming wouldn't make sense.